server/index.js

```diff
@@ -100,9 +100,11 @@ export default class Server {
       '/static/:path+': async (req, res, params) => {
         const p = join(this.dir, 'static', ...(params.path || []))
         await this.serveStatic(req, res, p)
-      },
+      }
+    }
 
-      '/:path*': async (req, res) => {
+    if (this.config.useFileSystemPageRoutes !== false) {
```

Comment:
Think we can just remove the `!== false` and add the default here https://github.com/zeit/next.js/blob/master/server/config.js#L6